### DISCUSSION TITLE
<A>rchival: (active) No plots found

### DISCUSSION THREAD

**Discussion by Salz0** [ericaltendorf/plotman#586] [Q&A] [answered]
<P>lotting: (active) <just started job> <A>rchival: (active) No plots found

Archive dirs free space
plots1:  100GB | plots2:  100GB |  |  |  |  |

So it sees in `plotman interactive` the disks, but above it says no plots found when there are -k25 plots in the final dst.

Is this because of my -k25? Can't load larger due to absence of space, but I want to make sure it all works

**Comment by altendky** [comment, marked as answer]
Duplicate of https://github.com/ericaltendorf/plotman/discussions/584.
